tests/foreman/ui/test_contenthost.py

```diff
@@ -51,9 +51,19 @@
     pytest.skip('skipping tests due to missing settings', allow_module_level=True)
 
 
+@pytest.fixture(scope='module', autouse=True)
```

Comment:
Why this is `autouse=True` and not on_demand, this would impact all the  tests in this module. Any reason why you would like to keep `host_details_ui` OFF ?

Reply:
host_details_ui needs to stay set to 'No' for all of the tests in this module, since they rely on the old host UI, and this is by far the easiest way to let them access it. I thought that autouse true was the easiest way to achieve this, rather than have each host set that property.

Reply:
Yeah, thats the very right way of doing it.